### PR TITLE
[Feat] ADF-834 E2E: properties with simple types

### PR DESCRIPTION
**Pull request by clagosarias**
https://oat-sa.atlassian.net/browse/ADF-834

Related PR: https://github.com/oat-sa/extension-tao-item/pull/552

Create a test to cycle through simple type properties and validate them

### CODE REVIEW

File: views/cypress/support/manageSchema.js [hunk 1]
```diff
@@ -29,6 +37,8 @@
  * @param {String} options.editUrl - url for the editing class POST request
  */
  Cypress.Commands.add('addPropertyToClass', (options) => {
+   options.propertyType = options.propertyType || 'list';
+   options.propertyListValue = options.propertyListValue || 'Boolean';
```
Comment (adriloma) [inline views/cypress/support/manageSchema.js:41]: there's no chance of expecting `''` or `0` on `propertyType ` or `propertyListValue` right? If those are expected value `OR` operator will return right-hand operand.

**Review by adriloma — APPROVED** [review]
Code review only. Looks good, just check a minor comment for a possible edge case.

- [x] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Pull request's target is not `master`
- [x] Commits are following conventional commits
- [x] Commits messages are meaningful
- [x] Commits are atomic

**Review by Silvia-loza — APPROVED** [review]
All new tests pass consistently.  Small issue with deleting in the `afterAll().`
- [X] New code is covered by tests (if applicable)
- [X] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [X] New code is respecting code style rules
- [X] New code is respecting best practices
- [X]  Pull request's target is not `master`
- [X] Feature is working correctly on my local machine (if applicable)
- [X] Acceptance criteria are respected
- [X] Pull request title and description are meaningful

File: views/cypress/support/manageSchema.js (hunk 1 above)
Comment (clagosarias) [reply views/cypress/support/manageSchema.js:41]: yes, this is the case 👍
